packages/chromaprint/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://acoustid.org/chromaprint
+TERMUX_PKG_DESCRIPTION="C library for generating audio fingerprints used by AcoustID"
+TERMUX_PKG_LICENSE="LGPL-2.1, MIT"
+TERMUX_PKG_VERSION=1.4.3
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL=https://github.com/acoustid/chromaprint/releases/download/v${TERMUX_PKG_VERSION}/chromaprint-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=ea18608b76fb88e0203b7d3e1833fb125ce9bb61efe22c6e169a50c52c457f82
+TERMUX_PKG_DEPENDS=ffmpeg
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DCMAKE_BUILD_TYPE=Release -DBUILD_TOOLS=ON -DBUILD_SHARED_LIBS=ON"
+termux_step_post_make_install() {
+  ln -sf "${TERMUX_PREFIX}/lib/libchromaprint.so" "${TERMUX_PREFIX}/lib/libchromaprint.so.${TERMUX_PKG_VERSION:0:1}"
+  ln -sf "${TERMUX_PREFIX}/lib/libchromaprint.so" "${TERMUX_PREFIX}/lib/libchromaprint.so.${TERMUX_PKG_VERSION}"
```

Comment:
Why you are creating versioned links here ?

Reply:
Because the normal build process only generates libchromaprint.so and some software using the package looks for versioned libraries (e.g. https://raw.githubusercontent.com/beetbox/pyacoustid/master/chromaprint.py, line 32). Also, other OSs like Ubuntu package versioned libraries (https://packages.ubuntu.com/disco/amd64/libchromaprint1/filelist).

Reply:
> using the package looks for versioned libraries

Ok.

> Also, other OSs like Ubuntu package versioned libraries

Usually this is not needed on Android and build system (cmake or autotools) automatically detects that.